package.json

```diff
@@ -10,7 +10,7 @@
     "build:dev": "rollup -c",
     "build:analyze": "rollup -c --environment ANALYZER:'analyzer'",
     "watch": "rollup -c -w",
-    "prepublish": "in-publish && yarn build || not-in-publish",
+    "prepublishOnly": "yarn build",
```

Comment:
I think `prepare` would be better here than `prepublishOnly`

Reply:
`build` should be run on `publish` and `install`? Because that's what `prepare` would do since it preserves the old deprecated behavior.

Reply:
Ah I was confused. You are right, prepublish is the right thing to use here.